### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by mohanson**

### CODE REVIEW

File: ckb-debugger-tests/src/bin/gr-child-script.rs [hunk 1]
```diff
@@ -102,29 +90,42 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
...
+    } else {
+        Some(combine_lock_script).pack()
+    };
     let combine_lock_witness = CombineLockWitness::new_builder()
         .index(Uint16::new_unchecked(0u16.to_le_bytes().to_vec().into()))
         .inner_witness(inner_witness)
         .script_config(config.clone())
         .build();
-
+    let lock_wrapper_witness = LockWrapperWitness::new_builder()
```
Comment (XuJiandong) [inline ckb-debugger-tests/src/bin/gr-child-script.rs:123]: simplify it with "create_lock_wrapper_witness".

File: ckb-debugger-tests/src/bin/gr-update.rs [hunk 2]
```diff
@@ -40,7 +53,14 @@ pub fn main() -> Result<(), Box<dyn std::error::Error>> {
     output_data.extend_from_slice(child_script_config.as_slice());
     repr_tx.tx.outputs_data[0] = JsonBytes::from_vec(output_data);
 
-    let witness_args = create_witness_args(&child_script_config, 0, &[Bytes::default()])?;
+    let combine_lock_witness =
```
Comment (XuJiandong) [inline ckb-debugger-tests/src/bin/gr-update.rs:56]: simplify it with "create_lock_wrapper_witness".

**Review by XuJiandong — CHANGES_REQUESTED** [review]

File: ckb-debugger-tests/src/bin/gr-child-script.rs (hunk 1 above)
Comment (mohanson) [reply ckb-debugger-tests/src/bin/gr-child-script.rs:123]: The `wrapped_script` and `config` is an optional value, which does not satisfy the definition of the function.

File: ckb-debugger-tests/src/bin/gr-update.rs (hunk 2 above)
Comment (mohanson) [reply ckb-debugger-tests/src/bin/gr-update.rs:56]: There is no need to pass `wrapped_script`

**Review by XuJiandong — APPROVED** [review]

**Pull request merged**